### ISSUE TITLE
Format file annotations

### ISSUE THREAD

**Issue by BraisGabin** [pinterest/ktlint#689]
This should be flagged:
```kotlin
@file:JvmName package foo.bar
```
and reformatted to
```kotlin
@file:JvmName

package foo.bar
```
I tried to create a PR myself to fix this but I can't find a neat way to handle all the cases.

The problem is that the file annotations are treated as normal annotations but they are different. They have its own rules in the [coding conventions](https://kotlinlang.org/docs/reference/coding-conventions.html#file-annotations).
